application/src/main/java/uk/nhs/hee/tis/revalidation/service/RecommendationServiceImpl.java

```diff
@@ -283,26 +284,37 @@ public boolean submitRecommendation(String recommendationId, String gmcNumber,
   public TraineeRecommendationRecordDto getLatestRecommendation(String gmcId) {
     log.info("Fetching latest recommendation info for GmcId: {}", gmcId);
     final var optionalDoctorsForDB = doctorsForDBRepository.findById(gmcId);
-    final var optionalRecommendation = recommendationRepository
-        .findFirstByGmcNumberOrderByActualSubmissionDateDesc(gmcId);
-    if (!optionalDoctorsForDB.isPresent()) {
-      throw new RecommendationException(format(DOCTOR_NOT_FOUND_MESSAGE, gmcId));
-    }
+    final var recommendations = recommendationRepository.findByGmcNumber(gmcId);
+    final var draftRecommendations = recommendations.stream().filter(
+        recommendation -> recommendation.getRecommendationStatus().name().equals("READY_TO_REVIEW"))
+        .collect(Collectors.toList());
+    //Check draft recommendation: if yes return draft else normal original flow
+    if (draftRecommendations.size() > 0) {
+      final var draftRecommendation = draftRecommendations.get(0);
```

Comment:
Do we assume to have only 1 draft recommendation for every doctor?
Could we analyse the Prod data to check if this assumption is true, please?

Reply:
Test is modified with 2 draft and 1 NOT_STARTED recommendation

Reply:
If there're multiple draft recommendations, may I ask how you know which one is the latest?

Reply:
We've check there're only 1 doctor with multiple(2) draft recommendations. And those 2 recommendations are exactly the same. We agreed to ignore it for the moment.